subsys/bluetooth/controller/ll_sw/ull_df.c

```diff
@@ -124,6 +124,25 @@ int ull_df_reset(void)
 {
 	int err;
 
+#if defined(CONFIG_BT_CTLR_DF_ADV_CTE_TX)
+	struct ll_adv_set *adv;
+	uint8_t handle;
+
+	/* Get the advertising set instance */
+	for (handle = 0U; handle < BT_CTLR_ADV_SET; handle++) {
+		adv = ull_adv_is_created_get(handle);
+		if (!adv) {
+			continue;
+		}
+
+		if (adv->df_cfg) {
+			adv->df_cfg->is_enabled = 0U;
+			ull_df_adv_cfg_release(adv->df_cfg);
+			adv->df_cfg = NULL;
+		}
```

Comment:
Set to `NULL` is sufficient as `lll_df_adv_cfg_pool` is `mem_init` in `init_reset` function.

```suggestion
		adv->df_cfg = NULL;
```

Reply:
Unnecessary code removed